hat/assets/js/apps/Iaso/domains/instances/actions.js

```diff
@@ -116,6 +116,14 @@ export const reAssignInstance = (currentInstance, payload) => dispatch => {
         });
 };
 
+/* Submission Creation workflow
+ *  1. this function call backend create Instance in DB
+ *  2. backend contact enketo to generate a Form page
+ *  3. backend return an url to enketo
+ *  4. this function redirect  to the Enketo service
+ *  5. After submission Enketo/Backend redirect to the submission detail page
+ *  See enketo/README.md for full details.
+ */
 export const createInstance = (currentForm, payload) => dispatch => {
```

Comment:
At some point we can use react-query for this no ? maybe on another ticket ?

Reply:
We should, I left it here like this because this PR was already getting quite big but if you want I can convert it now

Reply:
leave it like that , it should go away in next refactor